shell.nix

```diff
@@ -8,6 +8,9 @@ pkgs.mkShell {
     nettools
     niv
     gmp
+    docker-compose
```

Comment:
Since you provide a `docker-compose.yaml` it is good to also provider `docker-compose` binary.